src/backend/function/Fusion.Resources.Functions/Functions/ProfileSync.cs

```diff
@@ -22,6 +24,31 @@ public ProfileSync(IHttpClientFactory httpClientFactory)
             resourcesClient = httpClientFactory.CreateClient(HttpClientNames.Application.Resources);
         }
 
+        [FunctionName("profile-sync-event")]
+        public async Task SyncProfile(
+            [EventSubscriptionTrigger(HttpClientNames.Application.People, "subscriptions/persons", "resources-profile")] MessageContext message,
+            ILogger log)
+        {
+            log.LogInformation("Profile sync event received");
+            log.LogInformation(message.Event.Data);
+            var body = message.GetBody<PeopleSubscriptionEvent>();
+
+            if (body.Type != PeopleSubscriptionEventType.ProfileUpdated)
+                return;
+
+            var refreshResponse = await resourcesClient.PostAsJsonAsync($"resources/personnel/{body.Person.Mail}/refresh", new { });
+
+            if (refreshResponse.StatusCode == System.Net.HttpStatusCode.NotFound)
+            {
+                log.LogWarning($"Person with email '{body.Person.Mail}' not found in Resources");
+                return;
+            }
+
+            refreshResponse.EnsureSuccessStatusCode();
```

Comment:
This will not give us any info in the logs on what went wrong. A better way would be to log the content if the status code is nonsuccessfull...
```
if (!refreshResponse.IsSucce...) {
   var content = await resp.Content.ReadAsStringAsync();.
   log.LogError(content);
   resp.EnsureResp...();
}